test/character_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/character'
+require 'pry'
+
+class CharacterTest < Minitest::Test
+
+  def test_character_exist
+    kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+
+    assert_instance_of Character, kitt
+    assert_equal "KITT", kitt.name
+    assert_equal "William Daniels", kitt.actor
+    assert_equal 1000000, kitt.salary
+  end
```

Comment:
this is testing everything required, but generally we'll split the `assert_instance_of` and the initialized attributes into two separate tests